Specs/DateTime/English/DateTimeModel.json

```diff
@@ -7468,5 +7468,461 @@
         }
       }
     ]
+  },
+  {
+    "Input": "My vacation is from 10-1-2018-10-7-2018",

```

Comment:
Can you please add all the corresponding specs to EnglishOther? We need to make sure there is no break between DMY and MDY date formats.